drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -302,35 +305,84 @@ function PhilipsHueApi:get_rtype_by_rid(rtype, rid)
   )
 end
 
----@return HueResourceResponse<HueLightInfo>
-function PhilipsHueApi:get_lights() return self:get_all_reprs_for_rtype(HueDeviceTypes.LIGHT) --[[@as HueResourceResponse<HueLightInfo>]] end
+---@return HueResourceResponse<HueLightInfo>?
+---@return string? err nil on success
+function PhilipsHueApi:get_lights() return self:get_all_reprs_for_rtype(HueDeviceTypes.LIGHT) end
 
----@return HueResourceResponse<HueDeviceInfo>
-function PhilipsHueApi:get_devices() return self:get_all_reprs_for_rtype("device") --[[@as HueResourceResponse<HueDeviceInfo>]] end
+---@return HueResourceResponse<HueDeviceInfo>?
+---@return string? err nil on success
+function PhilipsHueApi:get_devices() return self:get_all_reprs_for_rtype("device") end
 
----@return HueResourceResponse<HueZigbeeInfo>
-function PhilipsHueApi:get_connectivity_status() return self:get_all_reprs_for_rtype("zigbee_connectivity") --[[@as HueResourceResponse<HueZigbeeInfo>]] end
+---@return HueResourceResponse<HueZigbeeInfo>?
+---@return string? err nil on success
+function PhilipsHueApi:get_connectivity_status() return self:get_all_reprs_for_rtype("zigbee_connectivity") end
 
 function PhilipsHueApi:get_rooms() return self:get_all_reprs_for_rtype("room") end
 
 ---@param light_resource_id string
----@return HueResourceResponse<HueLightInfo>
+---@return HueResourceResponse<HueLightInfo>?
+---@return string? err nil on success
 function PhilipsHueApi:get_light_by_id(light_resource_id)
-  return self:get_rtype_by_rid(HueDeviceTypes.LIGHT, light_resource_id) --[[@as HueResourceResponse<HueLightInfo>]]
+  return self:get_rtype_by_rid(HueDeviceTypes.LIGHT, light_resource_id)
 end
 
 ---@param hue_device_id string
----@return HueResourceResponse<HueDeviceInfo>
+---@return HueResourceResponse<HueDeviceInfo>?
+---@return string? err nil on success
 function PhilipsHueApi:get_device_by_id(hue_device_id)
-  return self:get_rtype_by_rid("device", hue_device_id) --[[@as HueResourceResponse<HueDeviceInfo>]]
+  return self:get_rtype_by_rid("device", hue_device_id)
 end
 
 ---@param zigbee_resource_id string
----@return HueResourceResponse<HueZigbeeInfo>
+---@return HueResourceResponse<HueZigbeeInfo>?
+---@return string? err nil on success
 function PhilipsHueApi:get_zigbee_connectivity_by_id(zigbee_resource_id)
-  return self:get_rtype_by_rid("zigbee_connectivity", zigbee_resource_id) --[[@as HueResourceResponse<HueZigbeeInfo>]]
+  return self:get_rtype_by_rid("zigbee_connectivity", zigbee_resource_id)
 end
 
+---@param contact_resource_id string
+---@return HueResourceResponse<HueContactInfo>?
+---@return string? err nil on success
+function PhilipsHueApi:get_contact_by_id(contact_resource_id)
+  return self:get_rtype_by_rid("contact", contact_resource_id)
```

Comment:
you've got constants defined for these strings already (except for "room")

Reply:
Good catch, thanks!

Reply:
Fixed on https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1359